Sources/Controls/ValidatorTextView-AppKit.swift

```diff
@@ -66,7 +66,7 @@ open class ValidatorTextView: NSTextView, ValidatorControl {
         
         if !validateOnFocusLossOnly && affectedCharRange.location != 0,
             let conditions = conditions,
-            (!shouldAllowViolation || conditions[0].shouldAllowViolation) {
+            (!shouldAllowViolation || !conditions[0].shouldAllowViolation) {
```

Comment:
Do you think it's worth guarding that the conditions aren't empty? (to avoid a potential index out of range crash). Might be one to discuss later if we want to handle it with a nice error message for the developer.